Bool laser_node_rdy